tools_docker/Debian_Uwsgi/supervisord_tracim.conf

```diff
@@ -9,13 +9,21 @@ logfile=/var/tracim/logs/supervisord.log ; (main log file;default $CWD/superviso
 pidfile=/var/run/supervisord.pid ; (supervisord pidfile;default supervisord.pid)
 childlogdir=/var/tracim/logs/            ; ('AUTO' child log dir, default $TEMP)
 
+
+[program:xvfb]
+command=Xvfb :99 -screen 0 1x1x16
+stdout_logfile=/dev/null
+redirect_stderr=true
+autostart=true
+autorestart=false
+
 ; email notifier (if async email notification is enabled)
 [program:tracim_mail_notifier]
 directory=/tracim/backend/
 command=python3 /tracim/backend/daemons/mail_notifier.py
-stdout_logfile =/var/tracim/logs/mail_notifier.log
+stdout_logfile=/var/tracim/logs/mail_notifier.log
 redirect_stderr=true
-autostart=false
+autostart=true
```

Comment:
Not sure to understand, this mean that  mail notifier is activated even if mail notification is disabled ?

Reply:
We start all the time this service and I think we not stop it if email is not activated (why ? I think its a mistake)

Reply:
This service was always started in the develop version of the entrypoint.sh script, so I didn't change this behavior. The optional service was/is `tracim_mail_fetcher` (only launched if the `REPLY_BY_EMAIL` environment variable is set).

Reply:
Ok, fix this here if this is not too complicated or create an issue about this problem.

Reply:
I can, but what should be the fix ? Starting `tracim_mail_notifier` only when `REPLY_BY_EMAIL` is set ?

Reply:
No, reply by email is another feature, you should start tracim_mail_notifier when email_notification is enabled (and also normally when notification is in async mode).

Reply:
Created issue #3142 for this.